sw/device/rom_exts/rom_ext.h

```diff
@@ -0,0 +1,16 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_ROM_EXTS_ROM_EXT_H_
+#define OPENTITAN_SW_DEVICE_ROM_EXTS_ROM_EXT_H_
+
+#include <stdnoreturn.h>
+
+noreturn void rom_ext_boot(void);
+
+#ifdef __cplusplus
+}  // extern "C"
+#endif  // __cplusplus
```

Comment:
I think you forgot to delete this?

Reply:
The Mask ROM also has the C++ guard, aren't both of these a subject to mocked unit tests? In which case they would need a guard.

I have moved the guard just before `#include <stdnoreturn.h>`.